lib/service_config/bolt_server_config.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'hocon'
+require 'service_config/base_config'
+require 'bolt/error'
+
+module BoltServer
```

Comment:
this is not a standard file path for this module

Reply:
I was suggesting a new directory for the puma config scripts not these configuration classes.

Reply:
Suggestions?

Reply:
ruby files in lib should always contain classes in standard format. So `BoltServer::Config` should be in `lib/bolt_server/config`.

However we shouldn't clutter up the top of the repo with a bunch of different puma config scripts. so we can move `plan_config.rb` and `puma_config.rb` to some directory. `config/plan.rb` `config/transport.rb` for example.

Reply:
Ah, sorry - that wasn't clear but makes a lot more sense.